include/deal.II/hp/dof_handler.h

```diff
@@ -182,7 +182,7 @@ namespace hp
      *
      * @ingroup Iterators
      */
-    typedef typename ActiveSelector::cell_iterator        cell_iterator;
+    typedef TriaIterator < dealii::DoFCellAccessor < DoFHandler < dim, spacedim >, false > >        cell_iterator;
```

Comment:
Alternatively (and maybe easier to achieve), how about changing line 120 above from
```
typedef dealii::internal::DoFHandler::Iterators<DoFHandler<dim,spacedim>, false> ActiveSelector;
```
to
```
typedef dealii::internal::DoFHandler::Iterators<hp::DoFHandler<dim,spacedim>, false> ActiveSelector;
```
?